doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,343 @@
+.. _pymapdl_docker:
+
+****************
+MAPDL and Docker
+****************
+
+You can run MAPDL within a container on any OS using Docker and
+connect to it via PyMAPDL.
+
+There are several situations in which it is advantageous to run MAPDL
+in a containerized environment such as Docker or Singularity:
+
+- Run in a consistent environment regardless of the host OS.
+- Portability and ease of install.
+- Large-scale cluster deployment using Kubernetes
+- Genuine app isolation through containerization
+
+In this case, you use your local Python installation to
+connect to this instance.
+
+.. graphviz::
+
+    digraph {
+      node [];
+      subgraph cluster_frontend {
+        label="*HOST*";
+        Python;
+      subgraph cluster_backend {
+        MAPDL;
+        label="*DOCKER*";
+        }
+      }
+      Python -> MAPDL
+      MAPDL -> Python
+    }
+
+Requirements
+============
+
+You need to have access to a Docker image with MAPDL in it.
+Visit :ref:`ref_make_container` for more information on how
+to create your own MAPDL docker image.
+
+Once you have created and uploaded your Docker image to a registry,
+you can start to pull and use the image in other devices.
+
+.. warning::
+
+   MAPDL Docker images are not allowed to be shared in
+   public or free-to-access repositories or registries.
+   Doing this you violate Ansys policy.
+
+
+
+Configure Docker to access a private GitHub registry
+----------------------------------------------------
+
+In case you have created a Docker image and upload it to a GitHub
+private repository, you must authorise your docker installation
+to access this private package by using a personal access
+token.
+Create a GitHub personal access token with ``packages read`` permissions
+according to `Creating a personal access token <gh_creating_pat_>`_
+
+Save that token to a file with:
+
+.. code::
+
+   echo MY_GITHUB_TOKEN_WITH_PACKAGE_READ_PERMISSION > GH_TOKEN.txt
+
+
+This lets you send the token to Docker without leaving the token value
+in your history. Next, authorize Docker to access this repository
+with:
+
+.. code::
+
+    GH_USERNAME=<my-github-username>
+    cat GH_TOKEN.txt | docker login ghcr.io -u $GH_USERNAME --password-stdin
+
+
+.. _run_an_mapdl_image:
+
+Run an MAPDL Docker image
+=========================
+
+You can now launch MAPDL from Docker using the command line, a
+`docker compose file <run_an_mapdl_image_using_docker_compose_>`_ 
+or a script that gather the commands given to the command line.
+
+Your Docker image should have a valid MAPDL license configuration.
+The easiest way is to have an environment variable :env:`ANSYSLMD_LICENSE_FILE`
+pointing to a valid license server. This environment variable can be already
+included in the Docker image. However this is not recommended because it could
+expose the license server if the Docker image get leaked.
+The recommended approach is to set that environment variable when running the
+container. 
```

Comment:
```suggestion
The easiest way is to have an environment variable, :env:`ANSYSLMD_LICENSE_FILE`,
pointing to a valid license server. This environment variable can be already
included in the Docker image. However, this is not recommended because it could
expose the license server if the Docker image isleaked.
The recommended approach is to set that environment variable when running the
container. 
```